requirements.txt

```diff
@@ -6,6 +6,7 @@ djangorestframework==3.11.1
 idna==2.10
 optional-django==0.3.0
 pkg-resources==0.0.0
+psycopg2==2.8.5
```

Comment:
did you manually add this here? or used `pip freeze`? (tip: you should always try to avoid changing requirements by hand)

Reply:
I used pip freeze